cli/api/daemon.go

```diff
@@ -876,7 +878,7 @@ func initMetricsClient() *metrics.Client {
 	return client
 }
 
-func (d *daemon) initFacade() *facade.Facade {
+func (d *daemon) initFacade() (*facade.Facade, error) {
```

Comment:
Well, I was following the pattern that seemed most common for the other initialization calls which was to let the caller, `startMaster`, issue the `Fatal()` call.